tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh

```diff
@@ -51,6 +52,20 @@ bazel test \
       -//tensorflow/contrib/... \
       -//tensorflow/lite/... \
       -//tensorflow/python/compiler/tensorrt/... \
-
-
+&& \
+bazel test \
+      --config=rocm \
+      -k \
+      --test_tag_filters=rocm_multi_gpu, \
```

Comment:
need to add `-no_rocm` here

Reply:
Is that strictly true?  `rocm_multi_gpu` is already specific to ROCm.  Doesn't the `--test_tag_filters` whitelist only the `nccl_manager_test` with this tag?  Perhaps I misunderstood what tags can do.

Reply:
think about a future scenario where you have multiple tests with `rocm_mutli_gpu` tag and one of them needs to be disabled for some reason....

Reply:
another way to think about it is that adding `no_rocm` tag should always disable that test from running on the CI....so all `bazel test` commands must filter those tests out

Reply:
Force pushed after rebase.  Added `-no_rocm` to test tag filter for the `rocm_multi_gpu` case.